guides/common/modules/con_prerequisites-for-configuring-project-with-keycloak-quarkus-authentication.adoc

```diff
@@ -1,8 +1,28 @@
 [id="prerequisites-for-configuring-{project-context}-with-keycloak-authentication_{context}"]
 = Prerequisites for configuring {Project} with {keycloak-quarkus} authentication
 
-* A {keycloak} account with administrative privileges.
-* A {keycloak} server that uses HTTPS instead of HTTP and has been initialized with the `--http-relative-path=/auth` context path.
+On your {ProjectServer}:
+
+* Install the packages required for registering a {keycloak} client:
++
+[options="nowrap", subs="verbatim,quotes,attributes"]
+----
+# {project-package-install} mod_auth_openidc keycloak-httpd-client-install python3-lxml
+----
+// python3-lxml is only needed on EL8 because of https://issues.redhat.com/browse/RHEL-31496
```

Comment:
As noted in https://github.com/theforeman/foreman-documentation/issues/3675, it also affects EL9 and that's the only platform we now run on. Perhaps updated it while you're at it?
```suggestion
// python3-lxml is only needed on EL9 because of https://issues.redhat.com/browse/RHEL-31496
```